stream-chat-android-compose/src/main/AndroidManifest.xml

```diff
@@ -19,8 +19,14 @@
     >
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"
+        android:maxSdkVersion="32" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"
+        android:maxSdkVersion="32" />
+
+    <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
+    <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
+    <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
```

Comment:
Appart of those permission, do we need to update anything in our [StreamFileProvider](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/StreamFileProvider.kt) and the [filepath](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-common/src/main/res/xml/stream_filepaths.xml) we support?
Couldn't we move those permissions to `UI-Common` artifact?

Reply:
I suppose we can move it since both XML & Compose use these permissions.

I didn't notice any issues with file picking during testing, so most likely we can keep `file-provider` config untouched